config/locales/diaspora/fr.yml

```diff
@@ -471,42 +471,42 @@ fr:
       can_comment_q: "Qui peut commenter ou aimer mon message privé ?"
-      can_reshare_a: "Personne. Il n'est pas permis de repartage un message privé. Cependant, les utilisateurs appartenant à cet aspect sont susceptibles de le copier et le coller."
+      can_reshare_a: "Personne. Il n'est pas permis de repartage un message privé. Cependant, les comptes appartenant à cet aspect sont susceptibles de le copier et le coller."
       can_reshare_q: "Qui peut repartager mes messages privés ?"
       see_comment_a: "Seuls les gens avec qui le message a été partagé (les gens qui sont dans les aspects sélectionnés par le rédacteur original) peuvent voir ses commentaires et les \"j'aime\". "
       see_comment_q: "Lorsque je commente ou que j'aime un message privé, qui peut le voir ?"
       title: "Messages privés"
-      who_sees_post_a: "Seuls les utilisateurs placés dans cet aspect et connectés à diaspora* peuvent voir votre message privé."
+      who_sees_post_a: "Seuls les personnes placées dans cet aspect et connectées à diaspora* peuvent voir votre message privé."
       who_sees_post_q: "Lorsque j'envoie un message à un aspect (c'est-à-dire un message privé), qui peut le voir ?"
     private_profiles:
       title: "Profils privés"
-      whats_in_profile_a: "La biographie, la localisation, le genre, et l'anniversaire. Ce sont les éléments dans le bas de la section de l'édition de la page de profil. Toutes ces informations sont optionnelles — c'est à vous de décider si vous les renseignez ou non.  Les utilisateurs connectés que vous avez ajoutés à vos aspects sont les seuls qui peuvent lire votre profil privé. Ils verront aussi les messages privés publiés dans les aspects dont ils font partie, mélangés avec vos publications publiques, lorsqu'ils visiteront votre page de profil."
+      whats_in_profile_a: "La biographie, la localisation, le genre, et l'anniversaire. Ce sont les éléments dans le bas de la section de l'édition de la page de profil. Toutes ces informations sont optionnelles — c'est à vous de décider si vous les renseignez ou non.  Les personnes connectées que vous avez ajoutés à vos aspects sont les seuls qui peuvent lire votre profil privé. Ils verront aussi les messages privés publiés dans les aspects dont ils font partie, mélangés avec vos publications publiques, lorsqu'ils visiteront votre page de profil."
       whats_in_profile_q: "Qu'est ce que mon profil privé ?"
-      who_sees_profile_a: "N'importe quel utilisateur connecté avec lequel vous partagez (ce qui signifie que vous l'avez ajouté dans l'un de vos aspects). Les gens qui vous suivent, mais que vous ne suivez pas, verrons uniquement vos informations publiques."
+      who_sees_profile_a: "N'importe quel personne connectée avec laquel vous partagez (ce qui signifie que vous l'avez ajouté dans l'un de vos aspects). Les gens qui vous suivent, mais que vous ne suivez pas, verrons uniquement vos informations publiques."
```

Comment:
I know, but I won’t create an account just for it. I just pointed out a few obvious mistake, so that Aldarone can fix them before starting a discussion on webtranslateit.